js-runner/README.md

```diff
@@ -0,0 +1,51 @@
+# @grain/js-runner
+
+The JavaScript runner for the Grain language.
+
+## Why
+
+Grain programs need to be instantiated in a specific way within JavaScript environments. By using `@grain/js-runner`, you can bootstrap the environment to run your Grain programs.
```

Comment:
```suggestion
This project makes it easier to get your Grain programs running in a JavaScript environment—either node or the browser. If your Grain code is compiled in unlinked mode (`--no-link`), your programs need to be run with `@grain/js-runner` in order for us to do module resolution.
```

@ospencer do you like this or have a better suggestion?

Reply:
Looks great! I'd tweak yours just a tiny bit:
```suggestion
This project makes it easier to get your Grain programs running in a JavaScript environment—either Node.js or the browser. If your Grain code is compiled in unlinked mode (`--no-link`), `@grain/js-runner` will perform module resolution and load all necessary modules.
```